Chap_API_Sync_Access.tex

```diff
@@ -255,12 +517,15 @@ \subsection{\code{PMIx_Get_nb}}
 %%%%
 \format
 
-\copySignature{PMIx_Get_nb}{1.0}{
-pmix_status_t \\
-PMIx_Get_nb(const pmix_proc_t *proc, const char key[], \\
-\hspace*{12\sigspace}const pmix_info_t info[], size_t ninfo, \\
-\hspace*{12\sigspace}pmix_value_cbfunc_t cbfunc, void *cbdata);
-}
+\versionMarker{1.0}
+\cspecificstart
+\begin{codepar}
+pmix_status_t
+PMIx_Get_nb(const pmix_proc_t *proc, const char key[],
+            const pmix_info_t info[], size_t ninfo,
+            pmix_value_cbfunc_t cbfunc, void *cbdata);
+\end{codepar}
+\cspecificend
```

Comment:
Editorial: I think we need to adjust this to use the `\copySignature` syntax used to support the Use Case appendix.